runtime/pom.xml

```diff
@@ -51,10 +51,20 @@
             <artifactId>api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```

Comment:
Runtime should not have lombok dep.

Reply:
I added it since there is a log line in the OscMain class. Should we remove this log line?

Reply:
I think so, I would avoid to added extra deps for just this.